frontend/src/v5/ui/controls/inputs/colorPicker/colorPicker.component.tsx

```diff
@@ -42,8 +45,11 @@ export const ColorPicker = ({ value: inputValue, defaultValue, onChange, disable
 	};
 
 	useEffect(() => {
-		if (!inputValue) return;
-		setValue(rgbGroupColorToHex(inputValue));
+		if (!isEmpty(inputValue)) {
+			setValue(rgbGroupColorToHex(inputValue));
+		} else {
+			setValue(defaults(rgbGroupColorToHex(inputValue || defaultValue), DEFAULT_VALUE));
+		}
```

Comment:
Correctly setting the value when only the opacity is set (i.e. no color). The else scope, in fact, uses a fallback value in case both `inputValue` and `defaultValue` are missing either, which is wrong in the no-color case.

Reply:
Correctly setting the value when only the opacity is set (i.e. no color). The else scope, in fact, uses a fallback value in case both inputValue and defaultValue are missing either, which is wrong in the no-color case.

But I think I can make this a little clearer